pkg/pqm4/Makefile

```diff
@@ -0,0 +1,12 @@
+PKG_NAME=pqm4
+PKG_URL=https://github.com/jowlo/pqm4.git
+PKG_VERSION=riot-port
```

Comment:
Please use a commit-id and not a tag name here. The CI server will choke at those at some point.